Resources/views/CRUD/edit_orm_many_to_one.html.twig

```diff
@@ -42,8 +42,11 @@ file that was distributed with this source code.
         {% endif %}
 
         <span id="field_actions_{{ id }}" class="field-actions">
+            {% set display_btn_list = sonata_admin.edit == 'list' and sonata_admin.field_description.associationadmin.hasRoute('list') and sonata_admin.field_description.associationadmin.isGranted('LIST') and btn_list %}
+            {% set display_btn_add = sonata_admin.edit != 'admin' and sonata_admin.field_description.associationadmin.hasRoute('create') and sonata_admin.field_description.associationadmin.isGranted('CREATE') and btn_add %}
+            {% if display_btn_list or display_btn_add %}
             <span class="btn-group">
```

Comment:
Could you take profit to put `div` instead of `span` and see result?

AFAIK, `.btn-group` should be on `div` element. http://getbootstrap.com/components/#btn-groups-single